specification/applicationinsights/resource-manager/Microsoft.Insights/preview/2020-02-02-preview/components_API.json

```diff
@@ -0,0 +1,637 @@
+          ],
+          "x-ms-enum": {
+            "name": "RequestSource",
+            "modelAsString": true
+          }
+        },
+        "InstrumentationKey": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Application Insights Instrumentation key. A read-only value that applications can use to identify the destination for all telemetry sent to Azure Application Insights. This value will be supplied upon construction of each new Application Insights component."
+        },
+        "CreationDate": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Creation Date for the Application Insights component, in ISO 8601 format.",
+          "format": "date-time"
+        },
+        "TenantId": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Azure Tenant Id."
+        },
+        "HockeyAppId": {
+          "type": "string",
+          "description": "The unique application ID created when a new application is added to HockeyApp, used for communications with HockeyApp.",
+          "externalDocs": {
+            "url": "https://aka.ms/ewjuel"
+          }
+        },
+        "HockeyAppToken": {
+          "type": "string",
+          "description": "Token used to authenticate communications with between Application Insights and HockeyApp.",
+          "readOnly": true,
+          "externalDocs": {
+            "url": "https://aka.ms/sls76s"
+          }
+        },
+        "provisioningState": {
+          "type": "string",
+          "description": "Current state of this component: whether or not is has been provisioned within the resource group it is defined. Users cannot change this value but are able to read from it. Values will include Succeeded, Deploying, Canceled, and Failed.",
+          "readOnly": true
+        },
+        "SamplingPercentage": {
+          "type": "number",
+          "format": "double",
+          "description": "Percentage of the data produced by the application being monitored that is being sampled for Application Insights telemetry.",
+          "externalDocs": {
+            "url": "https://aka.ms/sl0yqd"
+          }
+        },
+        "ConnectionString": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Application Insights component connection string."
+        },
+        "RetentionInDays": {
+          "type": "integer",
+          "description": "Retention period in days.",
+          "default": 90
+        },
+        "DisableIpMasking": {
+          "type": "boolean",
+          "description": "Disable IP masking."
+        },
+        "ImmediatePurgeDataOn30Days": {
+          "type": "boolean",
+          "description": "Purge data immediately after 30 days."
+        },
+        "WorkspaceResourceId": {
+          "type": "string",
+          "description": "ResourceId of the workspace which the data will be ingested to."
+        }
```

Comment:
what validation do you do that the user has access to ingest to this workspace?  Also, what "workspace" resource is this referring to?  Log analytics?  It could benefit from being more explicit.  There are a lot of "workspaces" in Azure.

Reply:
we have a linkedAccessValidation we do on the ARM manifest - to make sure that the user has permissions to the workspace

Reply:
Please update the description so it is clear that this is a log analytics workspace

Reply:
done.